### ISSUE TITLE
Compiling Fails on nested Annotations

### ISSUE THREAD

**Issue by psusen** [mtarld/symbok-bundle#52]
Example Docblock
```
/**
 * Customer
 *
 * @RecordChangeLoggable
 * @ApiResource
 * @ORM\Table(name="customers", indexes={@ORM\Index(name="salesRepEmployeeNumber", columns={"salesRepEmployeeNumber"})})
 * @ORM\Entity
 */
```
Stacktrace (json)
[scratch_1.txt](https://github.com/mtarld/symbok-bundle/files/5056646/scratch_1.txt)

I think its because it gets translated to this : 
```
/**
 * Customer
 *
 * @Surplex\PhpComponent\RecordChangeLoggerBundle\Components\Annotations\RecordChangeLoggable
 * @ApiPlatform\Core\Annotation\ApiResource
 * @Doctrine\ORM\Mapping\Table (name="customers", indexes={@ORM\Index (name="salesRepEmployeeNumber", columns={"salesRepEmployeeNumber"})})
 * @Doctrine\ORM\Mapping\Entity
 */
```

The nested `@ORM\Index` does not get replaced for the FQDN

**Comment by mtarld** [comment]
Hi! Sorry I've been kind of busy theses days, I'll have a look ASAP.

Thanks for reporting it 🙂

**Comment by mtarld** [comment]
The fix is merged and will be released in 2.2.1. Could you please give a try with `dev-master`?

**Comment by mtarld** [comment]
Gave some try a seems to work! Thanks!
